mappings/python3.csv

```diff
@@ -64,7 +64,7 @@ term,0,,
 factor,0,,
 power,0,,
 trailed_atom,0,,
-atom,1,Atoms,#ABDBF7
+atom,0,Atoms,#ABDBF7
```

Comment:
I know it "doesn't matter", but when all the other ignored rules are one way, we should keep it consistent